src/libraries/System.Runtime/tests/System.Runtime.Tests/System/Type/TypeTests.cs

```diff
@@ -513,6 +513,19 @@ public void GetTypeByName_ValidType_ReturnsExpected(string typeName, Type expect
             Assert.Equal(expectedType, Type.GetType(typeName.ToLower(), throwOnError: false, ignoreCase: true));
         }
 
+        public static IEnumerable<object[]> GetTypeByName_Invalid_ClrArguments()
```

Comment:
```suggestion
        public static IEnumerable<object[]> GetTypeByName_InvalidElementType()
```
I think this is a better description for this set of test cases.